examples/show_starfile.py

```diff
@@ -0,0 +1,83 @@
+import napari
+
+import mrcfile
+
+from grace.io import starfile, core
+from pathlib import Path
+
+#
+path_input = input("Input data path:")
+
+if len(path_input) < 1:
+    DATA_PATH = Path(
+        "/Users/bcostagomes/Documents/2dEM_images/annotation_dataset"
+    )
+else:
+    DATA_PATH = Path(path_input)
+
+
+IMAGE_PATH = DATA_PATH / "images"
+STAR_PATH = DATA_PATH / "detections"
+
+# Makes the grace files/directory from the starfile folder, for every starfile.
+starfile.mkdir_grace_from_star(STAR_PATH)
+
+# Gets list of available files in the image
+name_list = [f for f in IMAGE_PATH.iterdir() if f.is_file()]
+
+
+GRACE_PATH = DATA_PATH / "grace"
+
+# Runs only the first one
+# TODO: run a batch
+
+FILE_STEM = Path(name_list[0].stem)
+
+print(FILE_STEM)
+
+
+IMAGE = IMAGE_PATH / FILE_STEM.with_suffix(".mrc")
+DETECTION = GRACE_PATH / FILE_STEM.with_suffix(".grace")
+
+with mrcfile.open(IMAGE, "r") as mrc:
+    image_data = mrc.data.astype(int)
```

Comment:
I'd suggest using the pre-defined image readers from `grace.io.image_dataset`, otherwise the code is duplicated and potentially introduces errors. Mentioned in issue #148

Reply:
Here's how I implement the readers in `examples/show_data.py`. Hope this helps 🚀 
https://github.com/alan-turing-institute/grace/blob/e5b228bcdcc87f9ab6a19de93b624e94f7137404/examples/show_data.py#L7
&
https://github.com/alan-turing-institute/grace/blob/e5b228bcdcc87f9ab6a19de93b624e94f7137404/examples/show_data.py#L23